cmd/ebrelayer/main.go

```diff
@@ -189,6 +194,32 @@ func initConfig(cmd *cobra.Command) error {
 	return viper.BindPFlag(flags.FlagChainID, cmd.PersistentFlags().Lookup(flags.FlagChainID))
 }
 
+func replayEthereumCmd() *cobra.Command {
+	//nolint:lll
+	replayEthereumCmd := &cobra.Command{
+		Use:     "replayEthereum [tendermintNode] [web3Provider] [bridgeRegistryContractAddress] [validatorMoniker] [validatorMnemonic] [fromBlock] [toBlock] [sifFromBlock] [sifEndBlock]",
+		Short:   "replay missed ethereum events",
+		Args:    cobra.ExactArgs(9),
+		Example: "replayEthereum tcp://localhost:26657 ws://localhost:7545/ 0x30753E4A8aad7F8597332E813735Def5dD395028 validator mnemonic --chain-id=peggy",
+		RunE:    RunReplayEthereumCmd,
+	}
+
+	return replayEthereumCmd
+}
+
+func replayCosmosCmd() *cobra.Command {
```

Comment:
Why are there two commands?  Seems like if you have to tell replayEthereumCmd the start/end blocks for both chains, then replayEthereumCmd could do everything.

Reply:
We can combine the two commands into single one. It is clearer if we have seperate commands. Then you know if the message missed in Ethereum or Sifchain processed by replay script.

Reply:
No need to combine them.  It just wasn't clear to me why you needed starting and ending blocks for both chains.